### ISSUE TITLE
New contrastive PronType?

### ISSUE THREAD

**Issue by Stormur** [UniversalDependencies/docs#732]
Let's consider some pronouns/determiners like 

lat. _alius_, _alter_; it. _altro_; eng. _other_, _another_; gr. _άλλος_;  ...

Now, where it is annotated as a `PRON`/`DET` (but often it is considered an `ADJ`...) it seems that most agree about `PronType=Ind`. We are indeed using this value in our Latin treebanks too, but, personally, I would say more due to a lack of more fitting values than true conviction.

In fact, no word in the [examples](https://universaldependencies.org/u/feat/PronType.html#ind-indefinite-pronoun-determiner-numeral-or-adverb) covers the same area of _alter_: I think that this word is referring to something quite precise, and the precision is given by **the contrast** to something else. One out of many observation is that e.g. in Italian we can and often have to precise the definiteness of _altro_: _l'altro_ 'the other (one)' vs. _un altro_ 'an other (one)'. This is not allowed with demonstratives like _questo_ 'this (one)' or _quello_ 'that (one)', and some indefinites only allow the undefined article: _(una) qualche persona_ '(a) some person'.

So, what is the opinion of the community about the introduction of a `PronType=Contr`? Wouldn't it fill a void of current `PronType`s?

**Comment by dan-zeman** [comment]
For me this word was an adjective, although I admit that this is a bias caused by selected datasets and traditional grammars, and the meaning feels at least partially pronominal.

Would you propose to also include the word _same_ in the category, besides _other_?

If it is a new pronominal type at all, then I would label it `PronType=Con` because all other [prontypes](https://universaldependencies.org/u/feat/PronType.html) are three-letter strings.

**Comment by Stormur** [comment]
I don't think that _same_ would fit into the `PronType=Con` category, but this is another interesting issue, because now that you say that... as of now they are counted as `PronType=Emp`, but I have always been left wondering if this "sameness" is emphatic. The traditional Latin denomination for elements like _idem_ 'the same', _ipse_ '-self' is "determinatives". There are others like _totidem_ 'the same number of'. Maybe here just a change of denomination from `Emp` to `Det` would improve the situation?

In any case, another element I would consider in Latin is _uter_ 'which one of two? / the one of the two that / either one of two'. It has a sense of indefiniteness as an interrogative, but otherwise it seems to precise something through contrast.
